src/components/avatar-image.tsx

```diff
@@ -1,9 +1,8 @@
 import * as React from "react";
+import Gravatar from "react-gravatar";
 import styled from "styled-components";
 
-const defaultAvatarUrl = "/icons/avatar_default.svg";
-
-const StyledImg = styled.img`
+const StyledImg = styled(Gravatar)`
```

Comment:
```suggestion
const StyledGravatar = styled(Gravatar)`
```

(`X` → `StyledX` for improved communication)